### ISSUE TITLE
Progress bar

### ISSUE THREAD

**Issue by DmitryKa** [sv99/lsdreader#5]
Hi

Is it possible to add any kind of progress bar? Maybe just a text output when every n decondings finish. For understanding how long it will work. It's confusing otherwise.

**Comment by sv99** [comment]
How long?
Standart process some minits!

**Comment by DmitryKa** [comment]
yeah, it tooks about 5-10 minutes, but when user doesn't know about it, he wait only seconds. He runs it, see no output and asks - does it woкk?
